ico-apps/django-importer
# Enhancement/3 importation progress status

Fix #3 (see TODO)

# Changes
* Add a column to import log detail showing the progress (percentage) of a running task
* Update number of saved rows of a running task
* Enlarge input_file field

# How to test
* Import a not very small CSV file (users, monitoringsites...). 
* While importation is running, refresh the page to see the increment of percentage progress and saved rows.

![import_log_progress](https://user-images.githubusercontent.com/25174947/99073977-34340780-25b7-11eb-912e-40c66c6374c6.png)

# TODO
* This progress status does not work for surveys importation.
